src/main/java/org/apache/sysml/utils/Statistics.java

```diff
@@ -850,6 +876,13 @@ public static String display(int maxHeavyHitters)
 								 ((double)sparkBroadcast.longValue())*1e-9,
 								 ((double)sparkCollect.longValue())*1e-9));
 			}
+			if (psNumWorkers.intValue() > 0) {
+				sb.append(String.format("Paramserv func number of workers:\t%d.\n", psNumWorkers.intValue()));
+				sb.append(String.format("Paramserv func total gradients compute time:\t%.3f secs.\n", psTotalGradientsComputeTime.doubleValue() / 1000));
+				sb.append(String.format("Paramserv func total aggregation time:\t%.3f secs.\n", psTotalAggregationTime.doubleValue() / 1000));
+				sb.append(String.format("Paramserv func model broadcasting time:\t%.3f secs.\n", psModelBroadcastingTime.doubleValue() / 1000));
+				sb.append(String.format("Paramserv func total batch slicing time:\t%.10f secs.\n", psTotalBatchSlicingTime.doubleValue() / 1000));
```

Comment:
Please reduce the text to make the block aligned with the rest of the summary statistics. For example, there is the output of ASP 1 epoch, and disabled native BLAS:

```
Total elapsed time:             110.704 sec.
Total compilation time:         1.921 sec.
Total execution time:           108.784 sec.
Number of compiled MR Jobs:     0.
Number of executed MR Jobs:     0.
Cache hits (Mem, WB, FS, HDFS): 251452/0/0/2.
Cache writes (WB, FS, HDFS):    17906/0/0.
Cache times (ACQr/m, RLS, EXP): 0.836/0.047/0.441/0.000 sec.
HOP DAGs recompiled (PRED, SB): 0/3476.
HOP DAGs recompile time:        10.880 sec.
Functions recompiled:           1.
Functions recompile time:       0.009 sec.
Paramserv func number of workers:       79.
Paramserv func total gradients compute time:    1164.429 secs.
Paramserv func total aggregation time:  89.878 secs.
Paramserv func model broadcasting time: 1.261 secs.
Paramserv func total batch slicing time:        0.0180000000 secs.
Total JIT compile time:         36.785 sec.
Total JVM GC count:             27.
Total JVM GC time:              4.088 sec.
Heavy hitter instructions:
  #  Instruction               Time(s)  Count
  1  relu_backward             408.486   5211
  2  conv2d_bias_add           406.428   3662
  3  paramserv                  98.013      1
  4  conv2d_backward_filter     83.384   3474
  5  conv2d_backward_data       73.418   1737
  6  ba+*                       71.182  10610
  7  relu_maxpooling            43.432   3662
  8  +*                         33.852  20844
  9  *                          32.668  24431
 10  relu_maxpooling_backward   26.492   3474

```

For example, you can use 
* Paramserv total num workers
* Paramserv setup time (prog, data) 
* Paramserv gradient compute time
* Paramserv model broadcast time
* Paramserv batch indexing time

As mentioned above it would be awesome to include the setup time as two statistics for (1) creating the workers including program/function copy, and (2) data partitioning